components/slack/slack.app.mjs

```diff
@@ -404,7 +404,7 @@ export default {
       const params = {
         types,
         cursor,
-        limit: 10,
+        limit: 200,
```

Comment:
@js07 can we paginate through all channels and return the whole list, so the user never has to click "load more"? and to confirm, we'll probably have to rev the version for any action that uses the `channel` prop, right? (and the trigger too?) cc @dylburger

Reply:
Does Slack have a channel search API? Then we could have the prop use that instead of returning all channels and doing search client-side

Reply:
It looks like Slack has a [channel search API](https://api.slack.com/methods/admin.conversations.search#markdown), but it's limited Enterprise Grid organizations (and requires the `admin.conversations:read` scope).

Reply:
Ok so that's probably not a viable route right now then